CONTRIBUTING.md

```diff
@@ -1,7 +1,328 @@
-To contribute to the Void documentation, please read the following sections of the Void Handbook. Pull requests that do not meet the criteria described in these sections will not be merged. Please try to address as many of the criteria as possible before submitting a PR - that will improve the chances of your contribution being accepted quickly.
+# Contributing
 
-- [About this Handbook](https://docs.voidlinux.org/about/about-this-handbook.html)
-- [Style Guide](https://docs.voidlinux.org/contributing/void-docs/style-guide.html)
-- [Submitting Changes](https://docs.voidlinux.org/contributing/void-docs/submitting.html)
+To contribute to the Void documentation, please read the following. Pull
+requests that do not meet the criteria described below will not be merged. Try
+to address as many of the criteria as possible before submitting a PR - that
+will improve the chances of your contribution being accepted quickly.
```

Comment:
Would it be possible to mention `check.sh` here? I feel like discovering it below is a bit harder.

Reply:
Agreed. Also, sorry about not having updated SUMMARY.md - i've been commenting out linkcheck in `book.toml` due to the ongoing timeout issues. PR updated.

Reply:
Is it still happening for you? If you could run it with the debug variable to see if we can catch the website that's giving us issue, that would be great! It stopped happening for me :/

Reply:
i didn't notice it in the (copious!) debug output for the Travis job for this PR just now, but i'll keep an eye on things. :-)